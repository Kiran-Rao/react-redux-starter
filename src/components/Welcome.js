import React from 'react';

const Welcome = (props) => (
  <div>
    <div>Hello World!</div>
    <div>The count is: {props.count}</div>
  </div>
);

export default Welcome;
