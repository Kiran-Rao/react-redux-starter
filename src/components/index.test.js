import test from 'ava';
import React from 'react';
import render from 'react-test-renderer';

import Button from './Button';
import Welcome from './Welcome';

test('Button snapshot', t => {
  const tree = render.create(<Button title="aoeu" click={() => {}} />).toJSON();
  t.snapshot(tree);
});

test('Welcome snapshot', t => {
  const tree = render.create(<Welcome count={5} />).toJSON();
  t.snapshot(tree);
});
