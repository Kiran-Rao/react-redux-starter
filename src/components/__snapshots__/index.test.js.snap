// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button snapshot 1`] = `
<button
  onClick={[Function]}
>
  aoeu
</button>
`;

exports[`Welcome snapshot 1`] = `
<div>
  <div>
    Hello World!
  </div>
  <div>
    The count is: 
    5
  </div>
</div>
`;
